test/auspice_client_requests.json

```diff
@@ -88,13 +94,13 @@
   },
   {
     "name": "Check getDataset API for sidecar file tip-frequencies",
-    "url": "/charon/getDataset?prefix=/flu/seasonal/h3n2/ha/2y&type=tip-frequencies",
+    "url": "/charon/getDataset?prefix=/seasonal-flu/h3n2/ha/2y&type=tip-frequencies",
     "expectStatusCode": 200,
     "responseIsJson": true
   },
   {
     "name": "Fetch dataset using /fetch URLs (from S3 staging bucket)",
-    "url": "/charon/getDataset?prefix=/fetch/staging.nextstrain.org/flu_seasonal_h3n2_ha_2y.json",
+    "url": "/charon/getDataset?prefix=/fetch/staging.nextstrain.org/seasonal-flu_h3n2_ha_2y.json",
```

Comment:
This test is failing on GitHub actions (as well as the subsequent sidecar test) but passing for me locally. Maybe related to #832, but I'm running locally without a cache now and it still passes.

Reply:
The test failure was due to a 500 response which seemed odd. I triggered a [re-run](https://github.com/nextstrain/nextstrain.org/actions/runs/8746823001/attempts/2) which was successful. 500 is likely from an uncaught exception, but I don't have a good explanation and can't reproduce...

Reply:
More verbose (e.g. debug level) logging would help here. (Improving logging has been on our to-do list for a while now.)

Reply:
I just remembered that `test:ci` writes logs to a `server.log` file which is uploaded as an artifact even on failures. I went to check if there was anything about the 500 response, but the artifact [linked](https://github.com/nextstrain/nextstrain.org/actions/runs/8746823001/job/24004318517#step:12:24) from workflow logs does not work anymore. Not sure why, since our repo's retention period is set to 90 days 😕

I have a feeling the existing logs would have been useful, since this is present in the re-run's logs:

```
[verbose]	next.js handling request for "/zika@2022-01-01"
Getting (nextstrain) datasets for: prefix=staging/zika@2023-02-21
[warning]	Failed to fetch v2 main JSON: BadRequestError: This resource cannot handle versioned dataset requests (version descriptor requested: "2023-02-21")
[verbose]	Sending BadRequestError: This resource cannot handle versioned dataset requests (version descriptor requested: "2023-02-21") error as JSON
[verbose]	Sending BadRequestError: This resource cannot handle versioned dataset requests (version descriptor requested: "2023-02-21") error as JSON
[verbose]	Sending BadRequestError: This resource cannot handle versioned dataset requests (version descriptor requested: "2023-02-21") error as HTML with Express' default error handler
BadRequestError: This resource cannot handle versioned dataset requests (version descriptor requested: "2023-02-21")
    at CoreDataset.versionInfo (file:///home/runner/work/nextstrain.org/nextstrain.org/src/sources/models.js:195:13)
    at CoreDataset.versionInfo (file:///home/runner/work/nextstrain.org/nextstrain.org/src/sources/core.js:149:20)
    at DatasetSubresource.url (file:///home/runner/work/nextstrain.org/nextstrain.org/src/sources/models.js:244:39)
    at _exists (file:///home/runner/work/nextstrain.org/nextstrain.org/src/sources/models.js:280:44)
    at CoreDataset.exists (file:///home/runner/work/nextstrain.org/nextstrain.org/src/sources/models.js:292:19)
    at ifDatasetExists (file:///home/runner/work/nextstrain.org/nextstrain.org/src/endpoints/sources.js:78:35)
    at nextHandler (file:///home/runner/work/nextstrain.org/nextstrain.org/src/negotiate.js:66:20)
    at dispatchByContentType (file:///home/runner/work/nextstrain.org/nextstrain.org/src/negotiate.js:69:18)
    at wrappedHandler (file:///home/runner/work/nextstrain.org/nextstrain.org/src/async.js:328:21)
    at Layer.handle [as handle_request] (/home/runner/work/nextstrain.org/nextstrain.org/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/runner/work/nextstrain.org/nextstrain.org/node_modules/express/lib/router/route.js:137:13)
    at file:///home/runner/work/nextstrain.org/nextstrain.org/src/endpoints/sources.js:56:43
    at Layer.handle [as handle_request] (/home/runner/work/nextstrain.org/nextstrain.org/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/runner/work/nextstrain.org/nextstrain.org/node_modules/express/lib/router/route.js:137:13)
    at file:///home/runner/work/nextstrain.org/nextstrain.org/src/endpoints/sources.js:40:3
    at Layer.handle [as handle_request] (/home/runner/work/nextstrain.org/nextstrain.org/node_modules/express/lib/router/layer.js:95:5)
```

Reply:
Anyways we're good for this PR. Just something to keep an eye out for in the future

Reply:
Continuing in #836